extras/mainnet.json

```
@@ -152,5 +152,8 @@
         "QuoteHandler": "0x71E1cc2F7574C798ED893Ef04D55D1E573bE95B1",
         "LenderVaultFactory": "0x1874a08f7975b25944feb989bbaaa464f61ab3bc"
 
+    },
+    "paladin": {
+        "QuestBoardV1": "0xf0CeABf99Ddd591BbCC962596B228007eD4624Ae"
```

```suggestion
        "QuestBoardV2": "0xf0CeABf99Ddd591BbCC962596B228007eD4624Ae"
```

or at least that is what i understand from the docs ([ref](https://doc.paladin.vote/deployed-contracts/mainnet#quest)), that this is quest v2

---

Hmm @Kogaroshi is this correct, and the new contract you will deploy soon is quest v3?

---

Yes, the current Board is the V2, but the new ones coming are more like V2.1